### PR TITLE
Create disable-sslv3.md

### PR DESCRIPTION
**Pull request by skibz**

### CODE REVIEW

File: security/ssl/disable-sslv3.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+SSLv3 is nearly two decades old. Support, however, remains strong. Disabling support on your server will disallow clients from using this insecure protocol.
```
Comment (Johanndutoit) [inline security/ssl/disable-sslv3.md]: I'd add a small intro line just real quick on what SSLv3 is just to keep the flow when reading and not just jump into "this is old".

**Comment by Johanndutoit** [comment]
This looks great, see comment on my thought for just a quick intro line but happy to merge.

**Comment by skibz** [comment]
added some intro stuff :+1:

**Comment by Johanndutoit** [comment]
Awesome this is great

**Pull request merged**